2024-03-14 21:56:04,125 INFO    StreamThr :34452 [internal.py:wandb_internal():86] W&B internal server running at pid: 34452, started at: 2024-03-14 21:56:04.124920
2024-03-14 21:56:04,127 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status
2024-03-14 21:56:04,129 INFO    WriterThread:34452 [datastore.py:open_for_write():87] open: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\run-ojg9h4uj.wandb
2024-03-14 21:56:04,130 DEBUG   SenderThread:34452 [sender.py:send():378] send: header
2024-03-14 21:56:04,164 DEBUG   SenderThread:34452 [sender.py:send():378] send: run
2024-03-14 21:56:05,194 INFO    SenderThread:34452 [dir_watcher.py:__init__():211] watching files in: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files
2024-03-14 21:56:05,195 INFO    SenderThread:34452 [sender.py:_start_run_threads():1099] run started: ojg9h4uj with start time 1710424564.126919
2024-03-14 21:56:05,199 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: check_version
2024-03-14 21:56:05,199 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: check_version
2024-03-14 21:56:05,969 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: run_start
2024-03-14 21:56:06,005 DEBUG   HandlerThread:34452 [system_info.py:__init__():26] System info init
2024-03-14 21:56:06,006 DEBUG   HandlerThread:34452 [system_info.py:__init__():41] System info init done
2024-03-14 21:56:06,006 INFO    HandlerThread:34452 [system_monitor.py:start():194] Starting system monitor
2024-03-14 21:56:06,006 INFO    SystemMonitor:34452 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-03-14 21:56:06,006 INFO    HandlerThread:34452 [system_monitor.py:probe():214] Collecting system info
2024-03-14 21:56:06,012 INFO    SystemMonitor:34452 [interfaces.py:start():190] Started cpu monitoring
2024-03-14 21:56:06,013 INFO    SystemMonitor:34452 [interfaces.py:start():190] Started disk monitoring
2024-03-14 21:56:06,013 INFO    SystemMonitor:34452 [interfaces.py:start():190] Started gpu monitoring
2024-03-14 21:56:06,026 INFO    SystemMonitor:34452 [interfaces.py:start():190] Started memory monitoring
2024-03-14 21:56:06,038 INFO    SystemMonitor:34452 [interfaces.py:start():190] Started network monitoring
2024-03-14 21:56:06,049 DEBUG   HandlerThread:34452 [system_info.py:probe():150] Probing system
2024-03-14 21:56:06,051 DEBUG   HandlerThread:34452 [system_info.py:_probe_git():135] Probing git
2024-03-14 21:56:06,055 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:06,115 DEBUG   HandlerThread:34452 [system_info.py:_probe_git():143] Probing git done
2024-03-14 21:56:06,115 DEBUG   HandlerThread:34452 [system_info.py:probe():198] Probing system done
2024-03-14 21:56:06,115 DEBUG   HandlerThread:34452 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22621-SP0', 'python': '3.9.18', 'heartbeatAt': '2024-03-14T13:56:06.050202', 'startedAt': '2024-03-14T13:56:04.120011', 'docker': None, 'cuda': None, 'args': ('config/train_shakespeare_char.py',), 'state': 'running', 'program': 'C:\\Users\\ALIENWARE\\Desktop\\transformer\\nanoGPT\\train.py', 'codePathLocal': 'train.py', 'codePath': 'train.py', 'git': {'remote': 'git@github.com:project-monkey/nanoGPT.git', 'commit': '683ac7a1b8bfc7b32314ccc9f5319c9446e07c22'}, 'email': '2649706356@qq.com', 'root': 'C:/Users/ALIENWARE/Desktop/transformer/nanoGPT', 'host': 'DESKTOP-CT8AN10', 'username': 'ALIENWARE', 'executable': 'C:\\Users\\ALIENWARE\\.conda\\envs\\nanoGPT\\python.exe', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 2304.0, 'min': 0.0, 'max': 2304.0}, 'cpu_freq_per_core': [{'current': 2304.0, 'min': 0.0, 'max': 2304.0}], 'disk': {'/': {'total': 227.97643661499023, 'used': 216.70825576782227}}, 'gpu': 'NVIDIA GeForce RTX 3070 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3070 Laptop GPU', 'memory_total': 8589934592}], 'memory': {'total': 31.73318862915039}}
2024-03-14 21:56:06,115 INFO    HandlerThread:34452 [system_monitor.py:probe():224] Finished collecting system info
2024-03-14 21:56:06,115 INFO    HandlerThread:34452 [system_monitor.py:probe():227] Publishing system info
2024-03-14 21:56:06,115 DEBUG   HandlerThread:34452 [system_info.py:_save_conda():207] Saving list of conda packages installed into the current environment
2024-03-14 21:56:06,202 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\conda-environment.yaml
2024-03-14 21:56:08,072 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:09,808 DEBUG   HandlerThread:34452 [system_info.py:_save_conda():219] Saving conda packages done
2024-03-14 21:56:09,809 INFO    HandlerThread:34452 [system_monitor.py:probe():229] Finished publishing system info
2024-03-14 21:56:09,813 DEBUG   SenderThread:34452 [sender.py:send():378] send: files
2024-03-14 21:56:09,813 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-metadata.json with policy now
2024-03-14 21:56:09,814 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:09,938 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: python_packages
2024-03-14 21:56:09,938 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: python_packages
2024-03-14 21:56:09,938 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:56:09,939 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:56:09,940 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:56:10,092 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:10,226 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\conda-environment.yaml
2024-03-14 21:56:10,227 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-metadata.json
2024-03-14 21:56:10,227 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\requirements.txt
2024-03-14 21:56:10,651 DEBUG   SenderThread:34452 [sender.py:send():378] send: telemetry
2024-03-14 21:56:11,259 INFO    wandb-upload_0:34452 [upload_job.py:push():131] Uploaded file C:\Users\ALIENWARE\AppData\Local\Temp\tmp4te5cxs5wandb\jze09vl1-wandb-metadata.json
2024-03-14 21:56:12,116 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:13,254 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:14,140 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:14,900 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:15,265 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:16,154 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:18,173 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:19,936 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:20,192 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:22,209 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:23,336 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 21:56:23,337 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 21:56:23,337 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 21:56:23,339 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 21:56:24,229 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:24,318 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 21:56:24,949 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:56:24,965 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:56:24,965 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:56:25,312 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:25,321 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:26,248 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:27,338 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:28,262 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:29,353 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:30,278 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:30,732 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:31,368 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:32,298 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:33,389 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:34,317 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:35,408 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:36,398 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:36,429 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:37,428 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:37,429 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\config.yaml
2024-03-14 21:56:38,419 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:39,444 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:39,955 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:56:39,971 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:56:39,971 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:56:40,433 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:41,457 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:42,138 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:42,447 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:43,477 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:44,458 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:45,489 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:46,472 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:47,994 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:48,483 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:50,498 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:52,514 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:53,030 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:53,690 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 21:56:53,690 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 21:56:53,691 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 21:56:53,693 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 21:56:54,529 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:54,548 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 21:56:54,969 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:56:54,984 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:56:54,985 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:56:55,558 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:56,551 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:57,580 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:56:58,087 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:56:58,570 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:56:59,595 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:00,591 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:01,618 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:02,611 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:03,639 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:03,867 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:04,628 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:05,660 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:06,050 DEBUG   SystemMonitor:34452 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-03-14 21:57:06,052 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 21:57:06,711 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:07,695 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:08,729 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:09,655 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:09,712 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:09,975 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:57:09,990 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:57:09,990 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:57:10,745 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:11,729 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:12,768 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:13,750 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:14,787 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:14,804 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:15,762 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:16,808 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:18,821 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:19,832 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:20,833 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:22,846 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:24,657 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 21:57:24,658 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 21:57:24,658 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 21:57:24,674 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 21:57:24,824 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 21:57:24,864 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:24,920 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:24,978 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:57:24,994 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:57:24,994 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:57:25,833 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:26,881 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:27,849 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:28,901 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:29,859 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:30,732 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:30,925 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:31,878 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:32,946 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:33,900 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:34,968 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:35,923 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:36,065 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 21:57:36,065 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:37,047 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:37,946 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:39,060 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:39,965 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:39,981 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:57:39,995 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:57:39,995 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:57:41,081 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:41,543 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:41,986 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:43,098 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:44,008 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:45,117 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:46,022 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:46,891 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:47,135 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:48,033 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:49,158 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:51,176 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:51,919 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:53,200 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:54,997 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:57:55,003 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:57:55,004 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:57:55,215 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:55,806 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 21:57:55,807 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 21:57:55,807 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 21:57:55,809 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 21:57:56,090 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 21:57:57,235 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:57:57,727 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:57:58,105 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:57:59,253 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:00,130 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:01,273 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:02,149 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:02,743 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:03,290 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:04,163 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:05,300 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:06,073 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 21:58:06,183 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:07,372 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:07,764 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:08,201 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:09,391 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:09,998 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:58:10,014 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:58:10,015 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:58:10,217 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:11,407 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:12,235 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:12,787 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:13,420 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:14,246 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:15,435 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:16,263 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:17,453 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:18,151 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:18,276 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:19,467 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:21,482 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:23,187 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:23,509 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:25,014 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:58:25,030 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:58:25,031 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:58:25,523 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:27,153 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 21:58:27,154 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 21:58:27,154 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 21:58:27,170 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 21:58:27,350 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 21:58:27,544 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:28,241 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:28,364 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:29,568 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:30,379 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:31,584 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:32,397 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:33,292 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:33,605 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:34,423 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:35,627 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:36,084 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 21:58:36,443 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:37,707 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:38,386 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:38,460 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:39,726 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:40,027 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:58:40,042 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:58:40,043 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:58:40,476 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:41,740 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:42,490 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:43,505 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:43,759 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:44,511 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:45,780 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:46,532 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:47,799 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:48,549 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:48,812 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:49,819 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:50,563 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:51,832 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:53,853 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:53,855 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:55,032 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:58:55,047 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:58:55,048 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:58:55,879 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:57,895 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:58:59,218 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 21:58:59,219 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:58:59,220 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 21:58:59,220 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 21:58:59,234 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 21:58:59,652 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:58:59,652 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 21:58:59,913 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:01,665 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:01,937 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:03,681 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:03,954 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:04,693 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:05,695 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:05,974 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:06,086 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 21:59:07,713 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:08,052 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:09,734 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:09,944 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:10,044 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:59:10,060 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:59:10,061 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:59:10,072 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:11,755 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:12,091 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:13,771 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:14,112 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:15,200 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:15,789 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:16,127 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:17,801 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:18,140 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:19,820 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:20,156 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:20,489 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:21,837 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:22,172 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:24,188 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:25,032 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:59:25,063 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:59:25,064 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:59:26,205 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:26,426 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:28,221 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:30,246 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:31,458 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:32,257 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 21:59:32,257 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 21:59:32,258 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 21:59:32,263 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:32,271 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 21:59:32,932 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 21:59:33,941 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:34,280 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:35,960 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:36,099 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 21:59:36,293 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:36,878 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:37,983 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:38,369 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:40,009 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:40,035 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:59:40,082 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:59:40,083 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:59:40,387 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:42,034 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:42,205 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:42,409 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:44,057 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:44,428 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:46,081 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:46,455 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:47,532 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:48,098 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:48,466 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:50,118 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:50,484 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:52,135 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:52,500 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:52,864 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 21:59:54,154 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 21:59:54,518 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:55,051 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 21:59:55,082 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 21:59:55,083 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 21:59:56,537 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:58,557 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 21:59:58,601 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:00,595 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:02,621 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:03,639 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:04,641 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:06,107 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:00:06,241 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:00:06,242 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:00:06,242 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:00:06,257 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:00:06,319 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:00:06,659 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:08,337 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:08,755 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:09,288 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:10,054 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:00:10,084 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:00:10,085 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:00:10,358 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:10,776 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:12,377 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:12,795 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:14,394 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:14,805 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:14,828 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:16,414 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:16,853 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:18,443 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:18,874 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:20,318 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:20,462 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:20,892 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:22,477 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:22,920 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:24,505 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:24,932 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:25,063 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:00:25,095 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:00:25,095 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:00:25,415 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:26,519 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:26,950 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:28,536 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:28,966 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:30,550 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:30,594 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:30,991 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:33,007 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:35,031 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:35,628 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:36,109 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:00:37,049 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:39,125 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:40,078 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:00:40,103 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:00:40,104 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:00:40,613 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:00:40,614 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:00:40,614 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:00:40,614 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:00:40,656 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:00:40,705 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:41,142 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:42,677 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:43,154 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:44,699 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:45,177 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:46,144 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:46,717 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:47,191 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:48,731 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:49,209 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:50,755 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:51,228 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:51,567 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:52,775 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:53,249 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:54,793 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:55,073 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:00:55,104 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:00:55,104 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:00:55,276 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:56,814 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:57,095 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:00:57,297 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:00:58,855 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:00:59,317 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:00,877 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:01,341 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:02,727 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:02,895 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:03,360 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:04,914 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:05,375 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:06,122 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:01:07,394 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:08,135 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:09,478 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:10,090 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:01:10,104 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:01:10,104 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:01:11,499 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:13,516 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:14,035 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:14,946 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:01:14,946 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:01:14,946 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:01:14,947 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:01:14,997 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:01:15,534 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:17,009 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:17,551 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:19,028 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:19,562 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:19,589 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:21,046 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:21,586 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:23,068 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:23,612 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:25,089 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:25,092 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:01:25,092 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:25,108 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:01:25,108 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:01:25,632 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:27,106 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:27,645 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:29,121 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:29,664 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:30,590 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:31,143 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:31,685 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:33,161 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:33,698 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:35,182 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:35,728 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:36,097 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:36,128 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:01:37,198 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:37,752 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:39,213 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:39,844 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:40,082 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:01:40,113 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:01:40,114 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:01:41,568 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:41,863 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:43,880 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:45,905 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:46,604 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:47,924 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:49,124 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:01:49,124 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:01:49,125 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:01:49,125 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:01:49,323 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:01:49,940 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:51,341 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:51,959 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:51,968 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:53,359 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:53,973 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:55,093 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:01:55,123 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:01:55,124 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:01:55,372 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:55,989 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:57,390 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:01:57,475 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:01:58,005 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:01:59,407 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:00,025 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:01,430 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:02,048 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:02,441 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:03,003 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:04,064 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:04,460 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:06,080 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:06,132 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:02:06,474 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:08,097 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:08,491 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:08,581 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:10,116 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:02:10,146 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:02:10,146 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:02:10,181 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:10,511 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:12,197 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:12,529 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:14,214 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:14,389 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:16,237 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:18,257 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:19,421 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:20,272 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:22,294 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:23,672 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:02:23,673 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:02:23,673 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:02:23,673 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:02:24,314 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:24,661 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:24,662 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:02:24,695 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:25,112 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:02:25,143 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:02:25,143 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:02:26,328 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:26,672 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:28,347 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:28,691 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:30,261 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:30,366 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:30,712 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:32,390 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:32,733 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:34,407 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:34,754 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:35,830 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:36,139 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:02:36,428 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:36,779 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:38,449 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:38,797 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:40,122 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:02:40,154 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:02:40,155 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:02:40,534 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:40,821 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:40,983 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:42,553 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:42,838 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:44,569 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:44,855 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:46,063 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:46,585 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:46,871 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:48,605 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:50,625 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:51,108 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:52,640 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:54,662 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:55,118 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:02:55,151 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:02:55,151 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:02:56,562 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:02:56,682 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:58,347 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:02:58,347 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:02:58,347 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:02:58,349 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:02:58,700 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:02:58,967 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:02:58,967 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:03:00,724 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:00,980 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:02,165 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:02,743 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:02,997 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:04,766 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:05,019 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:06,141 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:03:06,787 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:07,040 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:07,744 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:08,806 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:09,061 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:10,119 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:03:10,151 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:03:10,151 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:03:10,885 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:11,082 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:12,898 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:13,102 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:13,318 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:14,917 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:15,126 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:16,936 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:17,142 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:18,899 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:18,956 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:19,163 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:20,977 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:21,185 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:22,998 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:23,205 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:24,802 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:25,014 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:25,129 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:03:25,145 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:03:25,145 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:03:27,030 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:29,052 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:29,892 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:31,069 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:33,048 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:03:33,049 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:03:33,049 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:03:33,050 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:03:33,080 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:33,304 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:03:35,005 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:35,094 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:35,320 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:36,156 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:03:37,114 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:37,339 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:39,130 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:39,354 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:40,130 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:03:40,161 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:03:40,162 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:03:40,520 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:41,213 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:41,376 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:43,229 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:43,389 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:45,249 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:45,409 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:46,245 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:47,268 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:47,430 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:49,287 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:49,446 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:51,306 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:51,465 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:51,917 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:53,329 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:53,476 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:55,132 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:03:55,163 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:03:55,164 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:03:55,344 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:55,501 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:57,364 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:03:57,536 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:03:57,737 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:03:59,390 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:01,403 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:02,770 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:03,420 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:05,431 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:06,168 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:04:07,456 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:08,090 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:08,090 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:04:08,091 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:04:08,091 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:04:08,091 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:04:08,653 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:04:09,471 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:09,663 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:10,147 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:04:10,178 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:04:10,178 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:04:11,549 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:11,684 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:13,570 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:13,699 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:13,857 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:15,585 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:15,715 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:17,602 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:17,731 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:19,569 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:19,621 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:19,752 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:21,641 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:21,772 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:23,661 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:23,793 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:25,158 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:04:25,189 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:04:25,190 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:04:25,492 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:25,678 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:25,810 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:27,697 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:27,827 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:29,715 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:29,844 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:30,996 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:31,736 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:31,865 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:33,753 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:33,879 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:35,767 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:36,033 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:36,174 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:04:37,779 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:39,794 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:40,173 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:04:40,205 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:04:40,206 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:04:41,752 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:41,871 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:43,293 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:04:43,294 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:04:43,294 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:04:43,295 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:04:43,896 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:43,980 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:04:44,991 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:45,918 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:47,012 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:47,130 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:47,949 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:49,035 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:49,966 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:51,057 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:51,978 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:52,797 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:53,075 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:53,995 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:55,093 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:55,187 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:04:55,216 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:04:55,216 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:04:56,023 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:57,113 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:04:58,046 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:04:58,470 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:04:59,135 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:00,065 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:01,155 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:02,082 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:03,180 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:04,092 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:04,151 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:05,199 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:06,108 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:06,176 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:05:07,215 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:08,125 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:09,202 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:10,144 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:10,194 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:05:10,210 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:05:10,211 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:05:12,231 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:14,248 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:14,596 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:16,266 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:18,284 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:18,536 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:05:18,536 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:05:18,537 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:05:18,537 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:05:19,330 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:19,330 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:05:20,300 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:20,524 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:21,346 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:22,318 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:23,360 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:24,337 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:25,203 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:05:25,219 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:05:25,220 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:05:25,375 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:25,531 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:26,362 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:27,394 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:28,384 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:29,414 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:30,399 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:30,920 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:31,434 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:32,421 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:33,451 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:34,441 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:35,473 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:36,184 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:05:36,186 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:36,459 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:37,489 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:38,478 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:39,511 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:40,206 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:05:40,221 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:05:40,221 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:05:40,496 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:41,316 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:41,527 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:42,577 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:43,552 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:44,600 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:46,370 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:46,627 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:48,650 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:50,667 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:51,399 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:52,687 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:53,796 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:05:53,797 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:05:53,797 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:05:53,798 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:05:54,657 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:05:54,696 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:55,217 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:05:55,232 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:05:55,233 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:05:55,668 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:56,707 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:56,731 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:05:57,694 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:05:58,728 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:05:59,715 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:00,752 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:01,736 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:02,419 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:02,772 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:03,761 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:04,800 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:05,779 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:06,185 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:06:06,815 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:07,798 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:08,203 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:08,830 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:09,812 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:10,228 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:06:10,243 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:06:10,243 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:06:10,844 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:11,828 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:12,929 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:13,845 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:13,949 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:14,943 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:15,858 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:16,958 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:17,879 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:18,977 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:19,847 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:20,989 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:23,005 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:24,880 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:25,019 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:25,237 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:06:25,238 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:06:25,238 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:06:27,042 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:29,058 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:29,437 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:06:29,438 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:06:29,438 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:06:29,439 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:06:29,978 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:06:30,494 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:31,078 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:32,002 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:33,099 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:34,022 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:35,118 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:36,043 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:36,198 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:06:36,199 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:37,139 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:38,058 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:39,155 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:40,080 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:40,251 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:06:40,251 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:06:40,252 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:06:41,176 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:41,976 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:42,096 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:43,263 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:44,117 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:45,280 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:46,142 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:47,301 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:47,716 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:48,164 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:49,328 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:50,183 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:51,340 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:52,198 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:53,356 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:53,503 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:54,215 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:06:55,266 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:06:55,268 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:06:55,268 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:06:55,371 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:57,389 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:06:58,867 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:06:59,408 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:01,424 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:03,443 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:03,907 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:05,096 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:07:05,097 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:07:05,097 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:07:05,098 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:07:05,320 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:07:05,455 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:06,203 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:07:06,327 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:07,470 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:08,345 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:09,021 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:09,486 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:10,269 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:07:10,283 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:07:10,283 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:07:10,362 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:11,505 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:12,380 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:13,584 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:14,396 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:14,765 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:15,604 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:16,416 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:17,623 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:18,438 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:19,643 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:20,454 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:20,514 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:21,662 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:22,479 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:23,678 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:24,502 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:25,276 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:07:25,307 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:07:25,307 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:07:25,689 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:25,724 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:26,523 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:27,717 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:28,539 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:29,727 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:30,559 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:31,208 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:31,747 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:33,768 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:35,784 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:36,212 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:07:36,213 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:37,808 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:39,821 ERROR   gpu       :34452 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-14 22:07:40,291 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:07:40,305 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: stop_status
2024-03-14 22:07:40,306 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: stop_status
2024-03-14 22:07:40,766 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: partial_history
2024-03-14 22:07:40,767 DEBUG   SenderThread:34452 [sender.py:send():378] send: history
2024-03-14 22:07:40,767 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: summary_record
2024-03-14 22:07:40,768 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:07:40,874 DEBUG   SenderThread:34452 [sender.py:send():378] send: exit
2024-03-14 22:07:40,874 INFO    SenderThread:34452 [sender.py:send_exit():585] handling exit code: 0
2024-03-14 22:07:40,874 INFO    SenderThread:34452 [sender.py:send_exit():587] handling runtime: 694
2024-03-14 22:07:40,875 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:07:40,875 INFO    SenderThread:34452 [sender.py:send_exit():593] send defer
2024-03-14 22:07:40,875 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:40,875 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 0
2024-03-14 22:07:40,876 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:40,876 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 0
2024-03-14 22:07:40,876 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 1
2024-03-14 22:07:40,876 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:40,876 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 1
2024-03-14 22:07:40,876 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:40,876 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 1
2024-03-14 22:07:40,876 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 2
2024-03-14 22:07:40,876 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:40,876 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 2
2024-03-14 22:07:40,876 INFO    HandlerThread:34452 [system_monitor.py:finish():203] Stopping system monitor
2024-03-14 22:07:40,877 INFO    HandlerThread:34452 [interfaces.py:finish():202] Joined cpu monitor
2024-03-14 22:07:40,885 INFO    HandlerThread:34452 [interfaces.py:finish():202] Joined disk monitor
2024-03-14 22:07:40,885 DEBUG   SystemMonitor:34452 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-03-14 22:07:40,885 DEBUG   SystemMonitor:34452 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-03-14 22:07:40,930 INFO    HandlerThread:34452 [interfaces.py:finish():202] Joined gpu monitor
2024-03-14 22:07:40,930 INFO    HandlerThread:34452 [interfaces.py:finish():202] Joined memory monitor
2024-03-14 22:07:40,930 INFO    HandlerThread:34452 [interfaces.py:finish():202] Joined network monitor
2024-03-14 22:07:40,931 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:40,931 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 2
2024-03-14 22:07:40,931 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 3
2024-03-14 22:07:40,931 DEBUG   SenderThread:34452 [sender.py:send():378] send: stats
2024-03-14 22:07:40,931 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:40,931 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 3
2024-03-14 22:07:40,932 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:40,932 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 3
2024-03-14 22:07:40,932 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 4
2024-03-14 22:07:40,932 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:40,932 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 4
2024-03-14 22:07:40,932 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:40,932 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 4
2024-03-14 22:07:40,932 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 5
2024-03-14 22:07:40,932 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:40,932 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 5
2024-03-14 22:07:40,933 DEBUG   SenderThread:34452 [sender.py:send():378] send: summary
2024-03-14 22:07:40,933 INFO    SenderThread:34452 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-14 22:07:40,933 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:40,933 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 5
2024-03-14 22:07:40,933 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 6
2024-03-14 22:07:40,934 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:40,934 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 6
2024-03-14 22:07:40,934 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:40,934 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 6
2024-03-14 22:07:40,934 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 7
2024-03-14 22:07:40,934 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:40,934 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:40,934 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 7
2024-03-14 22:07:40,934 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:40,934 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 7
2024-03-14 22:07:41,658 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:07:41,720 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 8
2024-03-14 22:07:41,720 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:41,720 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 8
2024-03-14 22:07:41,722 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:41,722 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 8
2024-03-14 22:07:41,722 INFO    SenderThread:34452 [job_builder.py:build():298] Attempting to build job artifact
2024-03-14 22:07:41,722 INFO    SenderThread:34452 [job_builder.py:_get_source_type():428] is repo sourced job
2024-03-14 22:07:41,769 INFO    SenderThread:34452 [job_builder.py:build():404] adding wandb-job metadata file
2024-03-14 22:07:41,779 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 9
2024-03-14 22:07:41,779 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:41,779 DEBUG   SenderThread:34452 [sender.py:send():378] send: artifact
2024-03-14 22:07:41,779 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 9
2024-03-14 22:07:41,874 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-14 22:07:42,664 INFO    Thread-16 :34452 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:44,780 INFO    wandb-upload_0:34452 [upload_job.py:push():89] Uploaded file C:\Users\ALIENWARE\AppData\Local\wandb\wandb\artifacts\staging\tmpk1z40bor
2024-03-14 22:07:44,792 INFO    wandb-upload_1:34452 [upload_job.py:push():89] Uploaded file C:\Users\ALIENWARE\AppData\Local\wandb\wandb\artifacts\staging\tmpe2x45zd1
2024-03-14 22:07:46,901 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: keepalive
2024-03-14 22:07:46,961 INFO    SenderThread:34452 [sender.py:send_artifact():1443] sent artifact job-git_github.com_project-monkey_nanoGPT.git_train.py - {'id': 'QXJ0aWZhY3Q6NzU1NzIzOTg0', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjE0ODczODAzMA==', 'latestArtifact': None}}
2024-03-14 22:07:46,961 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:46,961 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 9
2024-03-14 22:07:46,961 INFO    SenderThread:34452 [dir_watcher.py:finish():358] shutting down directory watcher
2024-03-14 22:07:46,961 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:47,690 INFO    SenderThread:34452 [dir_watcher.py:finish():388] scan: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files
2024-03-14 22:07:47,690 INFO    SenderThread:34452 [dir_watcher.py:finish():402] scan save: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\conda-environment.yaml conda-environment.yaml
2024-03-14 22:07:47,691 INFO    SenderThread:34452 [dir_watcher.py:finish():402] scan save: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\config.yaml config.yaml
2024-03-14 22:07:47,691 INFO    SenderThread:34452 [dir_watcher.py:finish():402] scan save: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log output.log
2024-03-14 22:07:47,696 INFO    SenderThread:34452 [dir_watcher.py:finish():402] scan save: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\requirements.txt requirements.txt
2024-03-14 22:07:47,697 INFO    SenderThread:34452 [dir_watcher.py:finish():402] scan save: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-metadata.json wandb-metadata.json
2024-03-14 22:07:47,697 INFO    SenderThread:34452 [dir_watcher.py:finish():402] scan save: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json wandb-summary.json
2024-03-14 22:07:47,699 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 10
2024-03-14 22:07:47,699 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: poll_exit
2024-03-14 22:07:47,700 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:47,701 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 10
2024-03-14 22:07:47,702 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:47,703 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 10
2024-03-14 22:07:47,703 INFO    SenderThread:34452 [file_pusher.py:finish():172] shutting down file pusher
2024-03-14 22:07:47,906 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-14 22:07:47,907 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: poll_exit
2024-03-14 22:07:48,717 INFO    wandb-upload_1:34452 [upload_job.py:push():131] Uploaded file C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\config.yaml
2024-03-14 22:07:48,926 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-14 22:07:48,927 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: poll_exit
2024-03-14 22:07:49,932 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-14 22:07:49,933 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: poll_exit
2024-03-14 22:07:49,965 INFO    wandb-upload_2:34452 [upload_job.py:push():131] Uploaded file C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\output.log
2024-03-14 22:07:50,141 INFO    wandb-upload_0:34452 [upload_job.py:push():131] Uploaded file C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\conda-environment.yaml
2024-03-14 22:07:50,935 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-14 22:07:50,935 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: poll_exit
2024-03-14 22:07:51,317 INFO    wandb-upload_3:34452 [upload_job.py:push():131] Uploaded file C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\requirements.txt
2024-03-14 22:07:51,404 INFO    wandb-upload_4:34452 [upload_job.py:push():131] Uploaded file C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\files\wandb-summary.json
2024-03-14 22:07:51,618 INFO    Thread-15 :34452 [sender.py:transition_state():613] send defer: 11
2024-03-14 22:07:51,618 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:51,618 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 11
2024-03-14 22:07:51,619 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:51,619 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 11
2024-03-14 22:07:51,619 INFO    SenderThread:34452 [file_pusher.py:join():178] waiting for file pusher
2024-03-14 22:07:51,619 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 12
2024-03-14 22:07:51,620 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:51,620 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 12
2024-03-14 22:07:51,620 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:51,620 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 12
2024-03-14 22:07:51,620 INFO    SenderThread:34452 [file_stream.py:finish():595] file stream finish called
2024-03-14 22:07:51,942 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-14 22:07:53,496 INFO    SenderThread:34452 [file_stream.py:finish():599] file stream finish is done
2024-03-14 22:07:53,496 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 13
2024-03-14 22:07:53,496 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: poll_exit
2024-03-14 22:07:53,496 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:53,496 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 13
2024-03-14 22:07:53,497 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: status_report
2024-03-14 22:07:53,497 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:53,497 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 13
2024-03-14 22:07:53,497 INFO    SenderThread:34452 [sender.py:transition_state():613] send defer: 14
2024-03-14 22:07:53,497 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: defer
2024-03-14 22:07:53,497 INFO    HandlerThread:34452 [handler.py:handle_request_defer():172] handle defer: 14
2024-03-14 22:07:53,497 DEBUG   SenderThread:34452 [sender.py:send():378] send: final
2024-03-14 22:07:53,498 DEBUG   SenderThread:34452 [sender.py:send():378] send: footer
2024-03-14 22:07:53,498 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: defer
2024-03-14 22:07:53,498 INFO    SenderThread:34452 [sender.py:send_request_defer():609] handle sender defer: 14
2024-03-14 22:07:53,498 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-14 22:07:53,498 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-14 22:07:53,498 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: poll_exit
2024-03-14 22:07:53,498 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: server_info
2024-03-14 22:07:53,499 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: poll_exit
2024-03-14 22:07:53,499 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: get_summary
2024-03-14 22:07:53,499 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: server_info
2024-03-14 22:07:53,499 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: sampled_history
2024-03-14 22:07:53,500 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-14 22:07:53,501 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: job_info
2024-03-14 22:07:53,905 DEBUG   SenderThread:34452 [sender.py:send_request():405] send_request: job_info
2024-03-14 22:07:53,905 INFO    MainThread:34452 [wandb_run.py:_footer_history_summary_info():3851] rendering history
2024-03-14 22:07:53,905 INFO    MainThread:34452 [wandb_run.py:_footer_history_summary_info():3883] rendering summary
2024-03-14 22:07:53,907 INFO    MainThread:34452 [wandb_run.py:_footer_sync_info():3810] logging synced files
2024-03-14 22:07:53,908 DEBUG   HandlerThread:34452 [handler.py:handle_request():146] handle_request: shutdown
2024-03-14 22:07:53,908 INFO    HandlerThread:34452 [handler.py:finish():869] shutting down handler
2024-03-14 22:07:54,514 INFO    WriterThread:34452 [datastore.py:close():296] close: C:\Users\ALIENWARE\Desktop\transformer\nanoGPT\wandb\run-20240314_215604-ojg9h4uj\run-ojg9h4uj.wandb
2024-03-14 22:07:54,916 INFO    SenderThread:34452 [sender.py:finish():1521] shutting down sender
2024-03-14 22:07:54,916 INFO    SenderThread:34452 [file_pusher.py:finish():172] shutting down file pusher
2024-03-14 22:07:54,916 INFO    SenderThread:34452 [file_pusher.py:join():178] waiting for file pusher
